sql/bigistore.sql

```diff
@@ -17,6 +17,16 @@ RETURNS SETOF record
 AS 'istore','bigistore_each'
 LANGUAGE C STRICT IMMUTABLE;
 
+CREATE FUNCTION smallest_key(bigistore)
```

Comment:
how about naming these `first_key` and `last_key` ?

Reply:
It's how I initially named those functions but then changed the naming as I realized that `first`/`last` involves reasoning about `istore` ordering (it's always sorted, but one have to keep that in mind) while `smallest`/`biggest` is very explicit.

Reply:
mmh  `min_key`  `max_key` ?

Reply:
i like it